src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/DailyEntriesTest.kt

```diff
@@ -92,7 +92,6 @@ class DailyEntriesTest: AbstractTest() {
                 after = LocalDate.now().toString(),
                 vacation = false
             )
-
```

Comment:
A line break here between the vals and the assertions was perhaps helpful for readability